tests/openvino/test_quantization.py

```diff
@@ -221,17 +221,17 @@ class OVWeightCompressionTest(unittest.TestCase):
         ),
         (
             OVModelForCausalLM,
-            "opt",
+            "HuggingFaceH4/tiny-random-LlamaForCausalLM",
```

Comment:
you can either replace it with : 
```suggestion
            "llama",
```
or add a new `model_id` to [`MODEL_NAMES`](https://github.com/huggingface/optimum-intel/blob/a0dc06c84eefb969cfa52ba685a31c15b31f5b9c/tests/openvino/utils_tests.py#L58)

Reply:
Thanks! Unfortunately, AWQ can't be applied to `llama` (`fxmarty/tiny-llama-fast-tokenizer`) because the required patterns do not match. I've added a new model `llama_awq="HuggingFaceH4/tiny-random-LlamaForCausalLM"`.